### PR TITLE
Linux: Add terminator

### PR DESCRIPTION
**Pull request by core-man**
Related to #90.

### CODE REVIEW

File: source/computer/fedora-setup.rst [hunk 1]
```diff
@@ -192,6 +192,24 @@ Fedora 的归档管理器可以识别并打开 Linux 下的常见压缩格式（
 
     $ sudo dnf install unar
 
+终端
+^^^^^
+
+Fedora 自带的终端模拟器是 GNOME Terminal，使用起来中规中矩。
+日常科研经常需要开好几个终端，切换和管理起来比较麻烦。
+
+`Terminator <https://github.com/gnome-terminator/terminator>`__
```
Comment (seisman) [inline source/computer/fedora-setup.rst:201]:
```suggestion
`Terminator <https://gnome-terminator.org/>`__
```

Comment (seisman) [reply source/computer/fedora-setup.rst:201]: Also need a link of the documentation page (https://gnome-terminator.readthedocs.io/en/latest/).

Comment (core-man) [reply source/computer/fedora-setup.rst:201]: https://gnome-terminator.org/ is an older site. The readers also can find it on Github (right side)

Comment (seisman) [reply source/computer/fedora-setup.rst:201]: No. It's the latest site.

File: source/computer/fedora-setup.rst [hunk 2]
```diff
@@ -192,6 +192,24 @@ Fedora 的归档管理器可以识别并打开 Linux 下的常见压缩格式（
...
+Fedora 自带的终端模拟器是 GNOME Terminal，使用起来中规中矩。
+日常科研经常需要开好几个终端，切换和管理起来比较麻烦。
+
+`Terminator <https://gnome-terminator.org/>`__
+是一个功能强大的终端模拟器，最常用的功能应该是终端分割和终端切换。
+使用如下命令安装::
+
+    $ sudo dnf install terminator
+
+以下介绍几个常用快捷键:
```
Comment (seisman) [inline source/computer/fedora-setup.rst:207]:
```suggestion
以下介绍几个常用快捷键，详细用法见 `官方文档 <https://gnome-terminator.readthedocs.io/>`__：
```

**Comment by seisman** [comment]
Also add the same content to Ubuntu setup guide.

**Comment by core-man** [comment]
> Also add the same content to Ubuntu setup guide.

Will add it if you think this one is okay.

File: source/computer/fedora-setup.rst [hunk 3]
```diff
@@ -192,6 +192,24 @@ Fedora 的归档管理器可以识别并打开 Linux 下的常见压缩格式（
...
+Fedora 自带的终端模拟器是 GNOME Terminal，使用起来中规中矩。
+日常科研经常需要开好几个终端，切换和管理起来比较麻烦。
+
+`Terminator <https://gnome-terminator.org/>`__
+是一个功能强大的终端模拟器，最常用的功能应该是终端分割和终端切换。
+使用如下命令安装::
+
+    $ sudo dnf install terminator
+
+以下介绍几个常用快捷键，详细用法见 `官方文档 <https://gnome-terminator.readthedocs.io/>`__：
```
Comment (seisman) [inline source/computer/fedora-setup.rst:207]:
```suggestion
以下介绍几个常用快捷键，详细用法见\ `官方文档 <https://gnome-terminator.readthedocs.io/>`__：
```

**Review by seisman — APPROVED** [review]

**Review by seisman — APPROVED** [review]

**Pull request merged**